CODEOWNERS

```diff
@@ -3,6 +3,7 @@
 /mesonbuild/modules/cmake.py @mensinda
 /mesonbuild/modules/unstable_external_project.py @xclaesse
 /mesonbuild/modules/unstable_rust.py @dcbaker
+/mesonbuild/modules/unstable_wayland.py @markbolhuis
```

Comment:
What is Github supposed to do, error out with "error: not allowed to push updates to a file named CODEOWNERS because it has content that github doesn't like"?

In fact, github does at least do this:

> This CODEOWNERS file contains errors

This error shows up in git master too.

The file is useless, no one looks at it other than github and github makes it useless by not allowing component ownership by people who lack global push access.